### DISCUSSION TITLE
Could the `Anti-Features` be removed please? Could all features be `Modular add-on`s?

### DISCUSSION THREAD

**Discussion by rajibando** [T8RIN/ImageToolbox#1530] [Q&A]
Thank you for this app.

F-Droid repo has this caution:
> Anti-Features
>
> This app has features you may not like. [Learn more!](https://f-droid.org/en/docs/Anti-Features/)
>
>   This app promotes or depends entirely on a non-free network service
>  This app tracks and reports your activity

Could these anti-features be reconsidered?

Could a truncated version of the app without all the features be released?

This could be possible if all the features were made into modules that could be added or removed.

**Comment by T8RIN** [comment]
There are no such things, check #1491

**Reply by rajibando** [reply]
Will check and take up with F-Droid Mods.

The 'query' is actually 'Queries'. May please the rest part be addressed.

Postscript: [Yes, I checked Linsui's comments](https://github.com/T8RIN/ImageToolbox/issues/1491#issuecomment-2510538929). So good. ASA you post your update in F-Droid repo the flags would be gone. Great!

On my other queries: I only need a tiny self-sufficient image editor app for my smartphone to correct images crudely, by cropping, stretching and rotating as losslessly as possible.

Had your app been modular, I would have removed all the parts non-essential to me, and used only the bare-minimal, shredded app.

I already have GIMP and Inkscape installed in my system for more involved image editing. For smartphones only the said crude perspective corrections would suffice.

Thank you for responding to my message. Best wishes.

**Reply by T8RIN** [reply]
You can use ImageToolboxLite, but it's stripped and will not be maintained

**Reply by rajibando** [reply]
I searched `ImageToolboxLite` as well as `Image Toolbox Lite` in[ F-Droid](https://search.f-droid.org/?q=Image+Toolbox+Lite&lang=en) but it wasn't found, Mr. Mukhametzyanov.

**Reply by T8RIN** [reply]
It's not in fdroid, you can find it in GitHub only

**Reply by rajibando** [reply]
Please release it in F-Droid, Mr. Mukhametzyanov. It will be so kind of you! Please also fix a price on it., not a donation. If it satisfies me, I am willing to pay for it.
I also assure you based on my long experience: shredded, optimised, specialised software with small footprint have more demand than the generalised, heavy, high end ones. For example. I liked Word97 more than the later versions.
